apps/src/templates/teacherDashboard/teacherSectionsRedux.js

```diff
@@ -388,7 +395,8 @@ export const asyncLoadSectionData = id => dispatch => {
     '/dashboardapi/sections',
     `/dashboardapi/courses`,
     '/dashboardapi/sections/valid_scripts',
-    '/dashboardapi/sections/valid_course_offerings'
+    '/dashboardapi/sections/valid_course_offerings',
+    '/dashboardapi/sections/available_participant_types'
```

Comment:
nice job finding a way to make this request in parallel with the other requests 👍